accountsdb-plugin-manager/Cargo.toml

```diff
@@ -12,6 +12,7 @@ documentation = "https://docs.rs/solana-validator"
 [dependencies]
 bs58 = "0.4.0"
 crossbeam-channel = "0.5"
+json5 = "0.4.1"
 libloading = "0.7.2"
 log = "0.4.11"
 serde = "1.0.133"
```

Comment:
Have you seen if these serde dependencies can be removed?

Reply:
Indeed, `serde` is not required. Removed.